doc/content/the-things-stack/migrate-to-v3/migrate-using-console.md

```diff
@@ -5,7 +5,7 @@ weight: 100
 
 This section explains steps for migrating end devices from The Things Network V2 to The Things Stack Community Edition using <a href="https://www.thethingsindustries.com/docs/getting-started/console/" target="_blank">The Things Stack Community Edition Console</a>.
 
-{{< note >}} Migrating active sessions using The Things Stack Community Edition Console is available since the latest update to V3.12. <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/" target="_blank">click here</a>.{{</ note >}}
+{{< note >}} Migrating active sessions using The Things Stack Community Edition Console is available since the latest update to 3.12. <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/" target="_blank">Click here for more details</a>.{{</ note >}}
```

Comment:
```suggestion
{{< note >}} Migrating active sessions using The Things Stack Community Edition Console is available since the latest update to 3.12. <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/" target="_blank">See here for more details</a>.{{</ note >}}
```